apps/experiments/models.py

```diff
@@ -422,6 +423,20 @@ def get_api_url(self):
     def get_absolute_url(self):
         return reverse("experiments:single_experiment_home", args=[self.team.slug, self.id])
 
+    @property
+    def only_whitelisted_participants_allowed(self):
```

Comment:
nit naming:`is_public` and switch logic since at first read it's more intuitive

Reply:
Good point thanks

Reply:
https://github.com/dimagi/open-chat-studio/pull/625/commits/af8473ae0cea605473ed4794f1d5dc8848ed9754